src/olm/session/mod.rs

```diff
@@ -437,12 +439,15 @@ impl Session {
                 }
 
                 if let Some(chain) = pickle.sender_chains.get(0) {
+                    // XXX: Passing in secret array as value.
```

Comment:
Can we make these a bit more descriptive. If someone else comes along, they won't have a clue what those are about.